etc/helm/pachyderm/templates/pachd/deployment.yaml

```diff
@@ -325,13 +325,13 @@ spec:
             command:
             - /pachd
             - --readiness
+          failureThreshold: 10
+          timeoutSeconds: 30
         startupProbe:
           exec:
             command:
             - /pachd
             - --readiness
-          failureThreshold: 10
```

Comment:
We still need these.  Basically gives pachd 300 seconds to wait for postgres, apply migrations, and setup the config from the environment.